Library/Styles/ButtonStyles.swift

```diff
@@ -103,18 +103,13 @@ public let navyButtonStyle =
 
 public let saveButtonStyle =
   UIButton.lens.title(forState: .normal) .~ nil
-  <> UIButton.lens.tintColor .~ .ksr_green_800
-  <> UIButton.lens.image(forState: .normal) .~ image(named: "icon-heart-unfilled")
-  <> UIButton.lens.image(forState: .selected) .~ image(named: "heart")
+  <> UIButton.lens.image(forState: .normal) .~ image(named: "icon--heart-outline")
+  <> UIButton.lens.image(forState: .selected) .~ image(named: "icon--heart")
   <> UIButton.lens.accessibilityLabel %~ { _ in Strings.Save_this_project() }
 
 public let shareButtonStyle =
   UIButton.lens.title(forState: .normal) .~ nil
-  <> UIButton.lens.tintColor .~ .black
```

Comment:
This icon as well as the `x` icon (`icon--cross`) are still being rendered as black, instead of `dark_grey_900`. Is it possible to make all 3 `dark_grey_900`?

<img width="514" alt="screenshot 2018-01-04 12 41 01" src="https://user-images.githubusercontent.com/3104761/34576736-eeaf653e-f14c-11e7-91fe-53b2cb147461.png">